packages/contentful--cra-template-create-contentful-app/template/src/setupTests.ts

```diff
@@ -3,3 +3,8 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom';
+import { configure } from '@testing-library/react';
+
+configure({
+  testIdAttribute: 'data-test-id',
```

Comment:
Sorry for coming back to this PR even though it is already merged.

I thought about the CCA yesterday and I don't think that we should customize the default jest /testing-library behavior here and just leave the defaults. Yes, Contentful uses `data-test-id` for setting test ids on elements. But as this package is used by many users outside of Contentful, a change to a default is more likely to cause confusion and problems.

Reply:
it confused me when joining contentful, questioning if i forgot how to use getByTestId selector 😅

Reply:
wdyt @Chaoste ?

Reply:
Mmmh, I don't have a strong opinion. I guess my intention is to align our public repositories, namely "forma-36" and this one. For forma-36, we somehow "force" people to do this (if they need to reference our f36 components in their tests).

Reply:
I wasn't aware of how F36 assigns test-ids. So I think this is fine as we at least stay consistent with other Contentful repos.
Still unfortunate to change a default here...